docs/src/docs/security/security-assessment.md

```diff
@@ -1,5 +1,7 @@
 # Security Assessment
 
+**Assumption**: There is no central IRS catena-x service - IRS is meant to be run within one company. This assumption is crucial for this security assessment, because it reduces the attack surface significantly.
```

Comment:
@scherersebastian  this is correct. 
Is there need to verify that assumption in some sort?

Reply:
@jzbmw what do you mean by "verify"?
We can also remove the sentence, should be clear by now - hopefully.
For the end users it would also make more sense to add a similar sentence to the setUp / GettingStarted documentation.

Reply:
An assumption is something which can be checked for validity or is invalid. 
I think it is pretty clear that this is given, that the IRS runs in a company context.

Reply:
I removed the "outdated"/obsolete assumption